### ISSUE TITLE
Cannot evaluate on MSCOCO validation set 

### ISSUE THREAD

**Issue by philipperemy** [karpathy/neuraltalk2#82]
Hi there,
I'm new to Neural Talk.
I ran all the installation commands successfully.
I downloaded the validation 2014 from MSCOCO website.
Here is my command:
```
th eval.lua -model model_id1-501-1448236541.t7_cpu.t7 -image_folder val2014 -num_images 1 -gpuid -1
```
I'm using only the CPU. The t7 model file comes from the pretrained checkpoint link.

```
DataLoaderRaw loading images from folder: 	val2014	
listing all images in directory val2014	
DataLoaderRaw found 40504 images	
constructing clones inside the LanguageModel	
Killed
```

It seems everything is running up to a point where it prints ```Killed```. It turns out that when I execute it with ```sudo```, the script doesn't get killed but the same output is shown still no ```vis.json``` is created in ```vis/```.
Anybody has had the same problem?

I'm running on a Ubuntu 14.04.3 LTS 64 bits.

Many thanks in advance.

**Comment by philipperemy** [comment]
Just for people who are interested. The problem was that I didn't have enough RAM. I span up an instance with just 2GB and it was not clearly enough to load the model.

Solve the problem by adding more RAM.
